src/forms/cashflow.js

```diff
@@ -83,7 +85,7 @@ class CashFlowInputsRow extends Component {
     return (str) => {
       if (!isNonNegNumber(str)) {
         let message    = `That number doesn't look right`,
-            numPeriods = str.match(/\./g) || [];
+            numPeriods = str.match(PERIODS_REGEX) || [];
```

Comment:
The `num` prefix makes it sound like this variable is storing a count of periods, rather than the results of the match. We could do something like `numPeriods = (str.match(PERIODS_REGEX) || [undefined]).length` (although that's super ugly). Probably better to just name it something like `periodsInString`.

Reply:
True, I'll change it. Generally, though, I think variable name stuff isn't going to be the focus of these PRs.

Reply:
I think I'll just call it `periods` to simplify.